app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -724,4 +730,15 @@ public boolean wasAppEverInstalled(String packageName) {
         .map(downloads -> downloads != null && !downloads.isEmpty())
         .toSingle();
   }
+
+  public Completable uninstallApp(String packageName) {
+    return installer.uninstall(packageName);
+  }
+
+  public Single<Long> getInstalledAppSize(@Nullable String packageName) {
+    return installedRepository.getInstalled(packageName)
+        .first()
+        .toSingle()
+        .map(app -> app.getAppSize());
+  }
```

Comment:
I kept this here because InstallManager already had code relating InstalledApps (just like the method being used on Apps). Also since it manages installations (download + install), i figured it could make sense to have this code here as well, as it is getting the installations.